schema.sql

```diff
@@ -11,67 +11,93 @@ CREATE SCHEMA web3storage
     inserted_at timestamp with time zone DEFAULT timezone('utc'::text, now()) NOT NULL,
     updated_at timestamp with time zone
   )
-  CREATE TABLE auth_tokens (
+  CREATE TABLE auth_key (
     id BIGSERIAL PRIMARY KEY,
     name TEXT NOT NULL,
     secret TEXT NOT NULL,
     user_id bigint NOT NULL,
     inserted_at timestamp with time zone DEFAULT timezone('utc'::text, now()) NOT NULL,
     deleted_at timestamp with time zone
   )
-  CREATE TABLE uploads (
+  CREATE TYPE upload_type AS ENUM (
+    'Car',
+    'Blob',
+    'Multipart'
+  )
+  CREATE TABLE upload (
     id BIGSERIAL PRIMARY KEY,
     user_id bigint NOT NULL,
-    auth_token_id bigint,
+    auth_key_id bigint,
     content_id bigint NOT NULL,
     name TEXT,
+    type upload_type NOT NULL,
     inserted_at timestamp with time zone DEFAULT timezone('utc'::text, now()) NOT NULL,
     deleted_at timestamp with time zone DEFAULT timezone('utc'::text, now())
   )
-  CREATE TABLE contents (
+  CREATE TABLE content (
     id BIGSERIAL PRIMARY KEY,
     cid TEXT UNIQUE NOT NULL,
     dag_size bigint DEFAULT 0,
     inserted_at timestamp with time zone DEFAULT timezone('utc'::text, now()) NOT NULL,
     updated_at timestamp with time zone  DEFAULT timezone('utc'::text, now())
   )
-  CREATE TABLE pins (
+  CREATE TYPE pin_status AS ENUM (
+    'Undefined',
+    'ClusterError',
+    'PinError',
+    'UnpinError',
+    'Pinned',
+    'Pinning',
+    'Unpinning',
+    'Unpinned',
+    'Remote',
+    'PinQueued',
+    'UnpinQueued',
+    'Sharded'
+  )
+  CREATE TABLE pin (
     id BIGSERIAL PRIMARY KEY,
     content_id bigint NOT NULL,
     pin_location_id bigint NOT NULL,
-    status TEXT NOT NULL,
+    status pin_status NOT NULL,
     inserted_at timestamp with time zone DEFAULT timezone('utc'::text, now()) NOT NULL,
     updated_at timestamp with time zone DEFAULT timezone('utc'::text, now())
   )
-  CREATE TABLE pin_locations (
+  CREATE TABLE pin_location (
     id BIGSERIAL PRIMARY KEY,
     peer_id TEXT UNIQUE NOT NULL,
     peer_name TEXT,
     region TEXT
   )
-  CREATE TABLE aggregate_entries (
+  CREATE TABLE aggregate_entry (
     id BIGSERIAL PRIMARY KEY,
     content_id bigint NOT NULL,
     aggregate_id bigint NOT NULL,
     data_model_selector TEXT
   )
-  CREATE TABLE aggregates (
+  CREATE TABLE aggregate (
     id BIGSERIAL PRIMARY KEY,
-    aggregate_cid text UNIQUE NOT NULL, -- previously dataCid
+    data_cid TEXT UNIQUE NOT NULL,
```

Comment:
Keeping this as-is since it's Filecoin parlance and matches with `data_model_selector` (which is the selector used to extract data from the root `data_cid`).